src/test/rmc-dependency-test/diamond-dependency/run-dependency-test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+# Test for platform
+platform=`uname -sp`
+if [[ $platform != "Linux x86_64" ]]; then
+  echo "Codegen script only works on Linux x86 platform"
+  exit 0
+fi
+
+# Log output
+LOG="/tmp/dependency_test_log.txt"
+
+# Compile crates with RMC backend
+cd $(dirname $0)
+rm -rf build
```

Comment:
does `cargo rmc` not work

Reply:
it fails with:
```
rmc (dependencies) $ ./src/test/rmc-dependency-test/diamond-dependency/run-dependency-test.sh
ERROR: Unexpected number of json outputs: 5
```

It seems like the script for it assumes a single symbol table output, but here we get one per dependency and main.

Reply:
Lets file a ticket for that

Reply:
https://github.com/model-checking/rmc/issues/451